src/bitcoin-tx.cpp

```diff
@@ -687,18 +687,18 @@ static void MutateTx(CMutableTransaction& tx, const std::string& command,
     else if (command == "outaddr")
         MutateTxAddOutAddr(tx, commandVal);
     else if (command == "outpubkey") {
-        if (!ecc) { ecc.reset(new Secp256k1Init()); }
+        ecc.reset(new Secp256k1Init());
```

Comment:
1. The usage of `Secp256k1Init` is pretty obscure here. Why is it needed for those few commands but not others?
2. Would it be a better idea to move the Secp256k1Init into the functions that are called? As close as possible to where it is actually required, which is not hard to see now.

Pinging @sipa, the author of this.